experimental/addons/spark-kubernetes-operator/README.md

```diff
@@ -0,0 +1,170 @@
+| sparkConfigMap      | Specify the name of the ConfigMap containing Spark configuration files such as log4j.properties. The | string            | false    |         |
+|                     | controller will add environment variable SPARK_CONF_DIR to the path where the ConfigMap is mounted   |                   |          |         |
+|                     | to.                                                                                                  |                   |          |         |
+| hadoopConfigMap     | Specify the name of the ConfigMap containing Hadoop configuration files such as core-site.xml. The   | string            | false    |         |
+|                     | controller will add environment variable HADOOP_CONF_DIR to the path where the ConfigMap is mounted  |                   |          |         |
+|                     | to.                                                                                                  |                   |          |         |
++---------------------+------------------------------------------------------------------------------------------------------+-------------------+----------+---------+
+
+```
+
+# Example for how to run a component typed spark-cluster in application
+
+1. Firstly, copy the following example to "spark-app-v1.yaml":
+
+> The addon will create a namespace named "spark-cluster"
+
+```
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: spark-app-v1
+  namespace: spark-cluster
+spec:
+  components:
+  - name: my-spark-application-component
+    type: spark-application
+    properties:
+      name: my-spark-app
+      namespace: spark-cluster
+      type: Scala
+      mode: cluster
+      image: "gcr.io/spark-operator/spark:v3.1.1"
+      imagePullPolicy: Always
+      mainClass: org.apache.spark.examples.streaming.JavaQueueStream
+      mainApplicationFile: "local:///opt/spark/examples/jars/spark-examples_2.12-3.1.1.jar"
+      sparkVersion: "3.1.1"
+      driverCores: 1
+      executorCores: 1
+```
+
+2. Secondly, start the application:
+
+```
+vela up -f spark-app-v1.yaml
+```
+
+You will see the stdout like this:
+
+```
+Applying an application in vela K8s object format...
+I0227 16:54:37.069480  361176 apply.go:121] "creating object" name="spark-app-v1" resource="core.oam.dev/v1beta1, Kind=Application"
+✅ App has been deployed 🚀🚀🚀
+    Port forward: vela port-forward spark-app-v1 -n spark-cluster
+             SSH: vela exec spark-app-v1 -n spark-cluster
+         Logging: vela logs spark-app-v1 -n spark-cluster
+      App status: vela status spark-app-v1 -n spark-cluster
+        Endpoint: vela status spark-app-v1 -n spark-cluster --endpoint
+Application spark-cluster/spark-app-v1 applied.
+```
+
+3. Then, you can use the native command to check the status of the Spark applicaiton:
+
+```
+$ kubectl get sparkapplications -n spark-cluster
```

Comment:
We can use `vela status <app-name>` to show app's status.

Reply:
Good suggestion, will add this command to be a showcase.

Reply:
@wangyikewxgm Thanks for your suggestion, I have addressed it, please take a look when you have time.